src/test/java/resources/batch.feature

```diff
@@ -215,7 +205,38 @@ Feature: Batch Details API test
     Then I should get non empty response
     And I am able to parse batch transactions response
     And I fetch batch ID from batch transaction API's response
-    Then I will sleep for 5000 millisecond
+    Then I will sleep for 8000 millisecond
+    And I call the sub batch summary API for sub batch summary with expected status of 200
+    Then I am able to parse sub batch summary response
+    And I should assert total txn count and successful txn count in response
+
+  @gov @batch-teardown
+  Scenario: BD-015 Batch splitting test
+    Given I have tenant as "rhino"
+    And I have the demo csv file "ph-ee-bulk-splitting.csv"
+    And I create a new clientCorrelationId
+    And I have private key
+    And I generate signature
+    When I call the batch transactions endpoint with expected status of 202
+    Then I should get non empty response
+    And I am able to parse batch transactions response
+    And I fetch batch ID from batch transaction API's response
+    Then I will sleep for 8000 millisecond
+    When I call the batch summary API with expected status of 200
+    Then I am able to parse batch summary response
+    And Status of transaction is "COMPLETED"
+    And I should have matching total txn count and successful txn count in response
```

Comment:
add one more line here to assert number of sub batches > 1

Reply:
done